sda/cmd/api/swagger_v1.yml

```
@@ -0,0 +1,412 @@
+openapi: 3.0.3
+info:
+  title: SDA admin API
+  version: "1.0"
+  description: This is the admin API for the sensitive data archive.
+paths:
+  /c4gh-keys/add:
+    post:
+      description: Registers an crypt4gh public key hash in the database
```

```suggestion
      description: Registers an crypt4gh public key in the database
```

I think it's the key we register, right?

---

Ok, as I read the code, it's the key, not a hash.